Example/SmileID/HomeView.swift

```diff
@@ -27,7 +27,6 @@ struct HomeView: View {
                             name: "SmartSelfie™ Enrollment",
                             onClick: {
                                 viewModel.onProductClicked()
-                                smartSelfieEnrollmentUserId = generateUserId()
```

Comment:
Wouldn't this change mean that the delegate would no longer have the userId? i.e. on line 36?